src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -1273,27 +1275,12 @@ def aks_update(cmd,     # pylint: disable=too-many-statements,too-many-branches,
        not disable_ahub and \
        not enable_managed_identity and \
        not assign_identity:
-        raise CLIError('Please specify "--enable-cluster-autoscaler" or '
-                       '"--disable-cluster-autoscaler" or '
-                       '"--update-cluster-autoscaler" or '
-                       '"--cluster-autoscaler-profile" or '
-                       '"--enable-pod-security-policy" or '
-                       '"--disable-pod-security-policy" or '
-                       '"--api-server-authorized-ip-ranges" or '
-                       '"--attach-acr" or '
-                       '"--detach-acr" or '
-                       '"--uptime-sla" or '
-                       '"--load-balancer-managed-outbound-ip-count" or '
-                       '"--load-balancer-outbound-ips" or '
-                       '"--load-balancer-outbound-ip-prefixes" or '
-                       '"--enable-aad" or '
-                       '"--aad-tenant-id" or '
-                       '"--aad-admin-group-object-ids" or '
-                       '"--enable-ahub" or '
-                       '"--disable-ahub" or'
-                       '"--enable-managed-identity"')
-
-    instance = client.get(resource_group_name, name)
+        if prompt_y_n("no arguments specicied would you like to reconcile?", default="n"):
+            #duplicating this worse than goto? 
+            headers = get_aks_custom_headers(aks_custom_headers)
+            return sdk_no_wait(no_wait, client.create_or_update, resource_group_name, name, instance, custom_headers=headers)
```

Comment:
It seems that the change is made on a really really old code base, I suppose you need to update it to resolve some conflicts. Besides the function signature of `create_or_update` has changed since we adopted the track 2 SDK, thus you need to change the name of the argument `custom_headers` to `headers`.